src/com/oltpbenchmark/benchmarks/tpcc/ddls/tpcc-postgres-ddl.sql

```diff
@@ -144,20 +144,3 @@ CREATE TABLE warehouse (
 --add constraints and indexes
 CREATE INDEX idx_customer_name ON customer (c_w_id,c_d_id,c_last,c_first);
 CREATE INDEX idx_order ON oorder (o_w_id,o_d_id,o_c_id,o_id);
```

Comment:
We can remove this index since there is also a unique index on the same set of columns. So, let's just have 1 index on oorder table like this:
`CREATE UNIQUE INDEX idx_order ON oorder ((o_w_id,o_d_id) HASH, o_c_id, o_id DESC);`
Note that change to use `DESC` for `o_id` since that helps with the workload queries.

Reply:
Done.